game/graphics/renderer.cpp

```diff
@@ -81,6 +81,7 @@ void Renderer::resetSwapchain() {
   smpN.setClamping(ClampMode::ClampToEdge);
 
   sceneLinear    = device.attachment(TextureFormat::R11G11B10UF,w,h);
+  sceneTonemapped = device.attachment(TextureFormat::RGBA8, w, h);
```

Comment:
RenderTargets tend to take lot of gpu-memory - need to avoid allocation, if FXAA is not enabled

Reply:
added optional creation